webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -0,0 +1,225 @@
+// @ts-check
+
+const { basePath, env } = require("../utils/env")
+const { getCSSManifest } = require("../utils/getCSSManifest")
+const { HashedModuleIdsPlugin } = require("webpack")
+const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin")
+const path = require("path")
+const webpack = require("webpack")
+const WebpackManifestPlugin = require("webpack-manifest-plugin")
+
+export const clientNovoDevelopmentConfig = {
+  stats: "normal",
+  parallelism: 100,
+  mode: env.webpackDebug ? "development" : env.nodeEnv,
+  devtool: "source-map",
+  entry: {
+    "artsy-novo": [
+      path.resolve(process.cwd(), "src/novo/src/client.tsx"),
+    ],
+  },
```

Comment:
Do we need to duplicate all of this config? We should be able to use webpack smart merging to only replace the keys that change:

```
clientNovoDevelopmentConfig = {
   entry: {
     "artsy-novo": [
        path.resolve(process.cwd(), "src/novo/src/client.tsx"),
     ],
  },
}
```
Is (mostly) all that we'd need; and then the diff can be what we change here to help optimize bundle splitting. Same applies to the server config.

Reply:
I plan on cleaning up the WebPack config in a follow-up PR. `weback.smart` has been removed from the library due to many complex edge cases as the author put it and as such we should move away from using it and use `webpack.merge` instead.

Reply:
Duplication is preferable while novo is tweaked independently of force to simplify testing out changes without breaking the other app. A large amount of effort has been made to isolate novo code from force knowing that there will be temporary duplication.

Reply:
I understand the need for independence, but in practical terms we're duplicating 99.9% of the file, for both client and server. Is there any difference between novo webpack config and existing webpack config other than entrypoints?